controllers/admin/AdminImagesController.php

```diff
@@ -111,6 +117,14 @@ public function __construct()
                         'cast' => 'intval',
                         'type' => 'text',
                     ],
+                    'PS_WEBP_QUALITY' => [
+                        'title' => $this->trans('WEBP compression', [], 'Admin.Design.Feature'),
+                        'hint' => $this->trans('Ranges from 0 (worst quality, smallest file) to 100 (best quality, biggest file).', [], 'Admin.Design.Help') . ' ' . $this->trans('Recommended: 80.', [], 'Admin.Design.Help'),
```

Comment:
It seems that 'Recommended: 80." is new wording. Do you think it would be possible to replace "80" with a variable? That would avoid translators having to translate several strings several times. 
![image](https://user-images.githubusercontent.com/70583503/122354467-5167f480-cf51-11eb-803d-f0c37f57137c.png)

Reply:
Fixed